tests/python/unittest/test_operator.py

```diff
@@ -5158,6 +5158,80 @@ def check_squeeze_op(shape, axis=None):
     test = mx.sym.squeeze(data, axis=(2, 4))
     check_numeric_gradient(test, [data_tmp])
 
+def py_adaptive_avg_pool(x, height, width):
```

Comment:
Considering this method is only being used by one test, please define this inside test_adaptive_avg_pool_op to avoid confusion.

Reply:
Done. 👍